src/azure-cli/azure/cli/command_modules/keyvault/_command_type.py

```diff
@@ -113,7 +113,7 @@ def keyvault_command_handler(command_args):
                 # apply results transform if specified
                 transform_result = merged_kwargs.get('transform', None)
                 if transform_result:
-                    return _encode_hex(transform_result(result, **command_args))
+                    return _encode_hex(transform_result(result, **{**command_args, **{'kv_transform': True}}))
```

Comment:
I think it could be simplified:
```suggestion
                    return _encode_hex(transform_result(result, **{**command_args, 'kv_transform': True}))
```